homeassistant_api/models/domains.py

```diff
@@ -93,193 +105,249 @@ def __getattr__(self, attr: str):
-    NUMBER = 'number'
-    TEXT = 'text'
-    SEARCH = 'search'
-    TEL = 'tel'
-    URL = 'url'
-    EMAIL = 'email'
-    PASSWORD = 'password'
-    DATE = 'date'
-    MONTH = 'month'
-    WEEK = 'week'
-    TIME = 'time'
-    DATETIME_LOCAL = 'datetime-local'
-    COLOR = 'color'
+    NUMBER = "number"
+    TEXT = "text"
+    SEARCH = "search"
+    TEL = "tel"
+    URL = "url"
+    EMAIL = "email"
+    PASSWORD = "password"
+    DATE = "date"
+    MONTH = "month"
+    WEEK = "week"
+    TIME = "time"
+    DATETIME_LOCAL = "datetime-local"
+    COLOR = "color"
+
 
 # Selectors
 class ServiceFieldSelectorAction(BaseModel):
     optionsInSidebar: Optional[bool] = None
 
+
 class ServiceFieldSelectorAddon(BaseModel):
     name: Optional[str] = None
     slug: Optional[str] = None
 
+
 class ServiceFieldSelectorArea(BaseModel):
-    entity: Optional[Union[List[ServiceFieldSelectorEntityFilter], ServiceFieldSelectorEntityFilter]] = None
-    device: Optional[Union[List[ServiceFieldSelectorDeviceFilter], ServiceFieldSelectorDeviceFilter]] = None
+    entity: Optional[
+        Union[List[ServiceFieldSelectorEntityFilter], ServiceFieldSelectorEntityFilter]
+    ] = None
+    device: Optional[
+        Union[List[ServiceFieldSelectorDeviceFilter], ServiceFieldSelectorDeviceFilter]
+    ] = None
     multiple: Optional[bool] = None
 
+
 class ServiceFieldSelectorAreasDisplay(BaseModel):
     pass
 
+
 class ServiceFieldSelectorAttribute(BaseModel):
     entity_id: Optional[Union[List[str], str]] = None
     hide_attributes: Optional[List[str]] = None
 
+
 class ServiceFieldSelectorAssistPipeline(BaseModel):
     include_last_used: Optional[bool] = None
 
+
 class ServiceFieldSelectorBackground(BaseModel):
     original: Optional[bool] = None
     crop: Optional[CropOptions] = None
 
+
 class ServiceFieldSelectorBackupLocation(BaseModel):
     pass
 
+
 class ServiceFieldSelectorBoolean(BaseModel):
     pass
 
+
 class ServiceFieldSelectorButtonToggle(BaseModel):
     options: List[Union[str, ServiceFieldSelectorSelectOption]]
     translation_key: Optional[str]
     sort: Optional[bool]
 
+
 class ServiceFieldSelectorColorRGB(BaseModel):
     pass
 
+
 class ServiceFieldSelectorColorTemp(BaseModel):
     unit: Optional[str] = None
     min: Optional[number] = None
     max: Optional[number] = None
     min_mireds: Optional[number] = None
     max_mireds: Optional[number] = None
 
+
 class ServiceFieldSelectorCondition(BaseModel):
     optionsInSidebar: Optional[bool] = None
 
+
 class ServiceFieldSelectorConfigEntry(BaseModel):
     integration: Optional[str] = None
-  
+
+
 class ServiceFieldSelectorConstant(BaseModel):
     label: Optional[str] = None
     value: Union[str, number, bool]
     translation_key: Optional[str] = None
 
+
 class ServiceFieldSelectorConversationAgent(BaseModel):
-    language: Optional[str] = None # filtering by language not supported
+    language: Optional[str] = None  # filtering by language not supported
```

Comment:
Woops. Thats my comment I left. What I had on mind there is that afaik Hass does not externally expose conversation agent's languages.